### DISCUSSION TITLE
Faster keyword and tag creation

### DISCUSSION THREAD

**Discussion by mickey-yang** [ente-io/ente#1481] [Enhancements]
Thinking of Lightroom style keyword management. Currently we have to manually add #tags to act as keywords. Thinking about some ideas to speed up tagging and photo management.
Some of my major painpoints using existing tagging: 
- Need to individually tag each photo 
- Not sure if I'm using the right wording/spelling/phrasing to tag consistently

Some feature ideas:
- Store tags that have been used before, and be able to add them through a prepared list
- Tag management
- Filter photos by multiple tags
- Being able to multi-select photos and tag all of them
- Integration with "Keywords" field in metadata - this will enable integration with Lightroom keywords

Many thanks

**Comment by vishnukvmd** [comment]
Thanks for the feedback @mickey-yang! Will keep these in mind when we start working on tags (that is an improvement over the current feature for adding descriptions).

**Reply by jbelien** [reply]
Awesome news! Thanks @vishnukvmd! 👍

**Comment by jbelien** [comment]
Related to #3017

**Comment by eridian-blip** [comment]
Any updates on this feature? Not being able to bulk edit descriptions makes it extremely difficult to add meaningful tags

**Comment by mickey-yang** [comment]
I haven’t seen any related updates from the Ente team roadmaps. Personally
I have worked around it by “inverting” tagging - I make each tag the name
of an album and then bulk add photos multiple times to different albums,
which I prepend with “z” so it sorts at the end. It works for me, but also
looking forward to when we get tagging that is compatible with Lightroom
and stuff.
